doc/DeveloperGuide.md

```diff
@@ -94,6 +94,8 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be free for users.
+6. Should have different font colours to make it easier for user to see.
```

Comment:
Recall from our tutorial on why NFRs should not be too subjective. How do you quantify `different font colours`? Will you support 3 colours or #rrggbb?